newrelic/exporter_test.go

```diff
@@ -110,23 +110,23 @@ type Data struct {
 }
 
 type Common struct {
-	timestamp  interface{}       `json:"-"`
-	interval   interface{}       `json:"-"`
+	Timestamp  interface{}       `json:"-"`
+	Interval   interface{}       `json:"-"`
```

Comment:
These fixes were from a `go vet` failure.
```
Run go vet ./...
# github.com/newrelic/opentelemetry-exporter-go/newrelic
newrelic/exporter_test.go:113:2: struct field timestamp has json tag but is not exported
newrelic/exporter_test.go:114:2: struct field interval has json tag but is not exported
newrelic/exporter_test.go:122:2: struct field timestamp has json tag but is not exported
newrelic/exporter_test.go:129:2: struct field timestamp has json tag but is not exported
```
I didn't know if there was a reason that these were not exported originally, but I assume that since this is a test that it isn't a big deal. If this is used outside of the tests and causes a problem, let me know.

Reply:
I think instead we should just remove the `json:"-"` part probably.

Reply:
Good call!